packages/xrpc-server/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@atproto/xrpc-server",
   "version": "0.0.1",
-  "main": "src/index.ts",
+  "main": "dist/index.js",
```

Not sure what that was about!
```suggestion
  "main": "src/index.ts",
```